### DISCUSSION TITLE
Main script entry point to poetry package?

### DISCUSSION THREAD

**Discussion by metaperl** [python-poetry/poetry#6517] [Q&A] [answered]
Hello, I am developing a poetry package that I created with `poetry new flash-goban`

I plan to develop library code in `flash-goban/flash_goban` but I am not sure:

1. where to put `main.py` that will run my library code as a script
2. what poetry command to call to add `main.py` to my package.

**Comment by deadnews** [comment, marked as answer]
Example setup:
- `flash-goban/flash_goban/main.py`:
```py
def cli() -> None:
    pass
```
- `flash-goban/pyproject.toml`:
```toml
[tool.poetry.scripts]
flash-goban = "flash_goban.main:cli"
```
Docs: https://python-poetry.org/docs/pyproject/#scripts

Btw, it's not Poetry job to create users modules.
